### PR TITLE
Updates tk 2023 10 02

### PR DESCRIPTION
**Pull request by tomkralidis**

### CODE REVIEW

File: documentation/docs/practical-sessions/downloading-and-finding-data-from-wis2.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ title: Downloading and finding data from WIS2
 
 # Downloading and finding data from WIS2
 
+!!! abstract "Learning outcomes!"
+
+    By the end of this practical session, you will be able to:
+
+    - use pywis-pubsub to subscribe to a Global Broker and download data to your local system
```
Comment (david-i-berry) [inline documentation/docs/practical-sessions/downloading-and-finding-data-from-wis2.md:11]: Capitalise or lower case, both have been used and I've changed one of them to capitals.

File: documentation/docs/practical-sessions/initializing-wis2box.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ title: Initializing wis2box
 
 !!! abstract "Learning outcomes"
 
-    By the end of this session you will be able to:
+    By the end of this practical session, you will be able to:
 
     * Run the `wis2box-create-config.py` script to create the initial configuration
```
Comment (david-i-berry) [inline documentation/docs/practical-sessions/initializing-wis2box.md:11]: * or - for the list? I'm not sure it makes a difference but we use - elsewhere.

This is also an example of where we have capitalised the list items

File: documentation/docs/practical-sessions/querying-data-using-the-wis2box-api.md [hunk 3]
```diff
@@ -37,11 +46,17 @@ Click on the 'Browse' link, then click on the 'json' link.
 !!! question
     How many stations are returned? Compare this number to the station list in `/data/wis2box/metadata/station/station_list.csv` file when logged into wis2box (`python3 wis2box-ctl.py login`).
 
+??? success "Click to reveal answer"
```
Comment (david-i-berry) [inline documentation/docs/practical-sessions/querying-data-using-the-wis2box-api.md:49]: "Click to reveal answer" or "Click to reveal the answer"? I think we've used both and this is only a minor nitpick.

**Review by david-i-berry — CHANGES_REQUESTED** [review]
See comments on the individual files, mostly related to the lists:
- Do we capitalise the first letter of the items?
- or do we leave it in lower case?

Also do we want to use "Click to reveal answer" or "Click to reveal the answer"? Either work.

**Comment by tomkralidis** [comment]
Thanks @david-i-berry.  Consistency changes applied in latest commit.

**Review by david-i-berry — APPROVED** [review]

**Pull request merged**